department-of-veterans-affairs/caseflow
# Removed time slot validation when virtual days are 0

Resolves [CASEFLOW-1551](https://vajira.max.gov/browse/CASEFLOW-1551)

### Description
Removes the spreadsheet validations for time slot details if there are no virtual docket days when building the Regional Office hearing schedule

### Acceptance Criteria
- [ ] When the value for an RO's "# of Virtual Days" column is 0, none of the time slot columns are required
(i.e., "Number of Time Slots", "Length of Time Slots", and "Start Time" can be left blank without throwing errors)

### Testing Plan
1. Shadow user `BVATWARNER` and navigate to the build schedule page: http://localhost:3000/hearings/schedule/build
2. Click the `Upload Files` button
3. Set the start date to 04/01/2021 and the end date to 06/30/2021
4. Upload this file: [RO Assignment Test Q3 FY21 .xlsx](https://github.com/department-of-veterans-affairs/caseflow/files/6657626/RO.Assignment.Test.Q3.FY21.xlsx)
5. Ensure that the algorithm fails with the validation that RO37 is missing time slot details
6. Open the excel document and go to the last tab to change the requested virtual days for RO37 from 9 to 0
7. Upload the document again and ensure that the algorithm runs successfuly

### Code Documentation Updates
